python_training/XO.py

```diff
@@ -0,0 +1,181 @@
+import random
+
+PLAYER1_SYMBOL = "X"
+PLAYER2_SYMBOL = "O"
+
+
+class XOGame:
+    """
+    Creates a XO game for two players, in any wanted board size (n>2).
+    """
+
+    def __init__(self, player1, player2, board_size):
+        """
+        Initiates the XO game.
+
+        :param player1: The name of the first player
+        :param player2: The name of the second player
+        :param board_size: The length of the boards rows and columns.
+        """
+        # Create a mapping between the player representation and name
+        self.players = {0: player1, 1: player2}
```

Comment:
Since you have only two options, and you're using a number, you can switch to a list:
`self.players = [player1, player2]`
And then just use the index.